#!/bin/sh /usr/share/dpatch/dpatch-run

## 13_epgfix.dpatch by clocker at vdrportal.de
## http://www.vdrportal.de/board/thread.php?postid=246358#post246358
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Force saving EPG data on exit.

@DPATCH@
--- vdr-1.3.22-orig/vdr.c	2005-02-12 16:06:16.000000000 +0100
+++ vdr-1.3.22/vdr.c		2005-03-26 13:13:11.000000000 +0100
@@ -929,6 +929,7 @@
 Exit:
 
   PluginManager.StopPlugins();
+  cSchedules::Cleanup(true);
   cRecordControls::Shutdown();
   cCutter::Stop();
   delete Menu;
